pyalex/api.py

```diff
@@ -255,14 +278,6 @@ class BaseOpenAlex:
     def __init__(self, params=None):
         self.params = params
 
-    def _full_collection_name(self):
-        if self.params is not None and "q" in self.params.keys():
-            return (
-                f"{config.openalex_url}/autocomplete/{self.__class__.__name__.lower()}"
-            )
-        else:
-            return f"{config.openalex_url}/{self.__class__.__name__.lower()}"
-
```

Comment:
I refactored some things here to get everything to work. I prepared the API to deprecate `return_meta fully`. That raised some other issues.